app/components/team/Index.js

```diff
@@ -127,13 +127,15 @@ class TeamIndex extends React.PureComponent {
             <Panel.Header>
               Frequently Asked Team Questions
             </Panel.Header>
-            <Panel.Section className="max-width-2">
-              <h3 className="mt3 h4 bold">How do teams work with SSO?</h3>
-              <p>When a user signs in to SSO, the additional user is added to your account, and will be charged immediately, just as if you had invited them to the account.</p>
-              <h3 className="mt3 h4 bold">A question about teams?</h3>
-              <p>An answer about teams.</p>
-              <h3 className="mt3 h4 bold">A question about teams?</h3>
-              <p>An answer about teams.</p>
+            <Panel.Section className="max-width-3">
+              <h3 className="mt3 h4 bold">Will users (and API tokens) still have access to their pipelines?</h3>
+              <p>When you enable Teams we’ll create a default team called ”Everyone”, containing all your users and pipelines. This ensures that users, and their API tokens, will still have access to their pipelines.</p>
+              <h3 className="mt3 h4 bold">How does Teams work with SSO?</h3>
+              <p>When a user joins the organization via SSO, they’ll be automatically added to any teams that have the “Automatically add new users to this team” setting enabled.</p>
+              <h3 className="mt3 h4 bold">Can I delete the ”Everyone” team?</h3>
+              <p>You’re free to delete the default ”Everyone” team. To ensure uniterrupted access to pipelines we recommend creating new teams, and assigning access, before deleting the ”Everyone” team.</p>
```

Comment:
Could we make the first sentence something more direct? Like 'Yes you can delete or edit the everyone team'. Then it's clear right from the start?